rules/no-duplicate-literal-elements.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+const MESSAGE_ID = 'no-duplicate-literal-elements';
+const messages = {
+	[MESSAGE_ID]: 'Hardcoded `{{type}}` of `{{value}}` with duplicates.',
+};
+
+const arraySelector = [
+	'VariableDeclarator',
+	' > ',
+	'ArrayExpression',
+	' , ',
+	'ExpressionStatement',
+	' > ',
+	'ArrayExpression',
+	' , ',
+	'AssignmentExpression',
+	' > ',
+	'ArrayExpression',
+].join('');
```

Comment:
This is really bad... I didn't understand it at first.

```suggestion
const arraySelector = [
	'VariableDeclarator > ArrayExpression',
	'ExpressionStatement > ArrayExpression',
	'AssignmentExpression > ArrayExpression',
].join(',');
```

Reply:
This is really better, thanks.